stock_product_pack/models/stock_move_line.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    # TODO: Check if it is possible to remove it
```

Comment:
This TODO is half explained. Remove it doing what? Link if there's Odoo PR, or mention that the problem is in Odoo, etc.

Reply:
I think the explanation within the method is enough, do you need more? When I create a PR to solve this in odoo (it will take hours to review and try to solve the “real problem”) I will link it.